dashboard/src/app/(authenticated)/(diagram)/components/Diagram.tsx

```diff
@@ -49,7 +49,7 @@ export const Diagram: FC<Props> = ({ spec, wfRun }) => {
 
   const searchParams: ReadonlyURLSearchParams = useSearchParams()
   const threadRunNumberFromRedirection: number = Number(searchParams.get('threadRunNumber'))
-  const nodeRunNameToBeHighlighted: string = searchParams.get('nodeRunName')!
+  const nodeNameToBeHighlighted: string = searchParams.get('nodeName')!
```

Comment:
I'd prefer to keep the "Run" word here, because node and nodeRun are not the same thing :)

Reply:
True, good catch